src/components/tabs/tabs.jsx

```diff
@@ -0,0 +1,26 @@
+var classNames = require('classnames');
+var SubNavigation = require('../../components/subnavigation/subnavigation.jsx');
+var React = require('react');
+
+require('./tabs.scss');
+
+/**
+ * Container for a custom, horizontal list of navigation elements
+ * that can be displayed within a view or component.
+ */
+var Tabs = React.createClass({
```

Comment:
This is awesome – I could definitely use a tabs component in conference, so this is great to have!